packages/components/card/src/inline-entry-card/InlineEntryCard.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: 'InlineEntryCard'
+slug: /components/inline-entry-card/
+github: 'https://github.com/contentful/forma-36/tree/forma-v4/packages/components/inline-entry-card'
```

Comment:
This link is taking me to a 404

I believe it's because the path is actually something like `/components/card/src/inline-entry-card` or something